sys-libs/compiler-rt-sanitizers/compiler-rt-sanitizers-13.0.0.9999.ebuild

```diff
@@ -102,6 +102,7 @@ src_configure() {
 		local -x CC=${CHOST}-clang
 		local -x CXX=${CHOST}-clang++
 		strip-unsupported-flags
+		filter-flags '-D_FORTIFY_SOURCE*'
```

Comment:
Don't you need it for gcc too, though?

Reply:
After testing that, true, I had that modification for a while, I guess I got confused by `strip-unsupported-flags`.

Fixed it.